src/components/Explore/Explore.style.ts

```diff
@@ -356,3 +356,22 @@ export const NormalizeSubLabel = styled.div`
   margin-left: 2rem;
   transform: translateY(-0.75rem);
 `;
+
+export const DisclaimerWrapper = styled.div`
```

Comment:
these were originally in `components/Disclaimer/Disclaimer.style` and used by explore. theyre no longer used by the disclaimer so i moved them here